### PR TITLE
pdf-viewer: disable pdf-viewer window

### LINKED ISSUE #640: Disable PDF viewer
Can we disable the PDF viewer until it is fixed upstream?  Unless we want to do our own pdf viewer using pdf.js. 

Currently, it opens up the dialog to save, and also opens up a new window which is supposed to be the actual viewer, but it doesn't display the PDF.  Having to close the extra window is annoying.

### PR DESCRIPTION
**Pull request by vsvipul**
**What's this PR do?**
Disables pdf viewer as required in #640 ..
closes #640 

**You have tested this PR on:**
  - [ ] Windows
  - [x] Linux/Ubuntu
  - [ ] macOS

### CODE REVIEW

**Comment by akashnimare** [comment]
@vsvipul thanks for working on this. Instead of removing the code, maybe just comment it out since we'll use it at some point in future. @abhigyank what do you think?

**Comment by abhigyank** [comment]
Yeah, sounds right. Also add a comment as to the reason for commenting that part due to being a bug upstream.

**Comment by vsvipul** [comment]
@akashnimare @abhigyank i have commented out the code mentioning the bug-upstream. Please tell me if some changes are required.

File: app/renderer/js/utils/link-util.js [hunk 1]
```diff
@@ -35,11 +35,12 @@ class LinkUtil {
 		return isImageUrl.test(url);
 	}
 
-	isPDF(url) {
-		// test for pdf extension
-		const isPDFUrl = /\.(pdf)\?*.*$/i;
-		return isPDFUrl.test(url);
-	}
+	// Code to show pdf in a new BrowserWindow (currently commented out due to bug-upstream)
```
Comment (abhigyank) [inline app/renderer/js/utils/link-util.js]: Commenting this part of the code doesn't seem to be needed. Others are fine for me.

Comment (vsvipul) [reply app/renderer/js/utils/link-util.js]: done

**Review by abhigyank — APPROVED** [review]
Looks good to me @akashnimare .

**Comment by akashnimare** [comment]
Merged, thanks @abhigyank, @vsvipul 🚀

**Pull request merged**